<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthRegForm</class>
 <widget class="QWidget" name="AuthRegForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>465</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelLogin">
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditPass"/>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonAuth">
        <property name="text">
         <string>Auth</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonReg">
        <property name="text">
         <string>Reg</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonChange">
        <property name="text">
         <string>Change</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditLogin"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditPass_Conf"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelPass_Conf">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="errorLable">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
